no-nuxt-pls/components/NewsItem/NewsItem.vue

```diff
@@ -1,14 +1,42 @@
 <template>
   <div>
-    <h2>{{news.title}} ({{news.votes}})</h2>
+    <h2 class="text-xl text-center font-bold leading-7 text-gray-900 sm:text-2xl sm:truncate">{{news.title}} ({{news.votes}})</h2>
     <br>
-    <button @click="updateNews(1)">Upvote</button>
-    <button @click="updateNews(-1)">Downvote</button>
-    <button @click="deleteNews">Remove</button>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce consequat pulvinar convallis. Vivamus iaculis,
```

Comment:
Because you seem to be such a big fan of deeply nested npm dependencies: You know `lorem-ipsum`? https://github.com/knicklabs/lorem-ipsum.js#readme
Even more: https://github.com/templeman/awesome-ipsum